### DISCUSSION TITLE
Could some explain what limit/value I need to set for "MaxConnecs, AsyncIOThreadsCount and DiskWriteCacheSize"

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1342] [Q&A]
Hi,

I have a 1GB DDR4 RAM VPS what limit/value do I need to set for "MaxConnecs, AsyncIOThreadsCount and DiskWriteCacheSize" 
I would appreciate if someone could explain please?

**Comment by anasty17** [comment]
Max connection maybe 100 or 200 MaxConnecs overall
Asyncio threads number of cpu cores × 4
DiskWriteCacheSize 1/8 ram but make less better. like 32 mb

**Reply by erickyun** [reply]
Thanks
